client-v2/src/types/FaciaApi.js

```diff
@@ -0,0 +1,76 @@
+type FrontConfigResponse = {
+  collections: Array<string>,
+  priority?: PriorityName,
+  canonical?: string,
+  group?: string,
+  isHidden?: boolean,
+  isImageDisplayed?: boolean,
+  imageHeight?: number,
+  imageWidth?: number,
+  imageUrl?: string,
+  onPageDescription?: string,
+  description?: string,
+  title?: string,
+  webTitle?: string,
+  navSection?: string
+};
+
+type Platform = 'Web' | 'Platform';
+
+type CollectionConfigResponse = {
+  displayName: string,
+  type: string,
+  backfill?: Object,
+  href?: string,
+  groups?: Array<string>,
+  metadata?: Array<Object>,
+  platform?: string,
+  uneditable?: boolean,
+  showTags?: boolean,
+  hideKickers?: boolean,
+  excludedFromRss?: boolean,
+  description?: string,
+  showSections?: boolean,
+  showDateHeader?: boolean,
+  showLatestUpdate?: boolean,
+  excludeFromRss?: boolean,
+  hideShowMore?: boolean,
+  platform?: Platform
+};
+
+type FrontsConfigResponse = {
+  fronts: {
+    [string]: FrontConfigResponse
+  },
+  collections: {
+    [string]: CollectionConfigResponse
+  }
+};
+
+type FrontConfigResponseWithoutPriority = $Diff<
+  FrontConfigResponse,
+  { priority: PriorityName | void }
+>;
```

Comment:
Can we just `FrontConfigResponseWithoutPriority` here, we don't seem to be using anywhere.

Reply:
Agreed 👍